CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+include(ExternalProject)
+
+project(SDL LANGUAGES C)
+add_subdirectory(SDL2)
+
+add_subdirectory(sdl-gpu)
+target_link_libraries(SDL PUBLIC SDL_gpu)
+
+add_subdirectory(SDL_ttf)
+target_link_libraries(SDL PUBLIC SDL_ttf)
```

Comment:
as discussed, linking gpu and ttf publicly doesn't really make sense. But we cannot workaround this atm